subsys/bluetooth/controller/ll_sw/lll_filter.h

```diff
@@ -4,27 +4,31 @@
  * SPDX-License-Identifier: Apache-2.0
  */
 
-#if defined(CONFIG_BT_CTLR_FAL_SIZE)
-#define FAL_SIZE CONFIG_BT_CTLR_FAL_SIZE
+#if defined(CONFIG_BT_CTLR_FAL_SIZE) && defined(CONFIG_BT_CTLR_RL_SIZE)
+#define LLL_FILTER_SIZE MAX(CONFIG_BT_CTLR_FAL_SIZE, CONFIG_BT_CTLR_RL_SIZE)
+#elif defined(CONFIG_BT_CTLR_FAL_SIZE)
+#define LLL_FILTER_SIZE CONFIG_BT_CTLR_FAL_SIZE
+#elif defined(CONFIG_BT_CTLR_RL_SIZE)
+#define LLL_FILTER_SIZE CONFIG_BT_CTLR_RL_SIZE
 #else
-#define FAL_SIZE 8
-#endif /* CONFIG_BT_CTLR_FAL_SIZE */
+#define LLL_FILTER_SIZE 8
```

Comment:
Where this value comes from?

Reply:
I think it's assumed that the LLL filter has size of 8 if Kconfig does not specify it's size.
It used to be defined like that, so I left it as it was. If needed we may address it in a separate PR I think.